src/www/ui/template/ui-clearing-view.js.twig

```diff
@@ -42,7 +42,7 @@ selectedLicensesTableConfig = {
 
 editableConfiguration = {
   "sUpdateURL": "?mod=conclude-license&do=updateClearings",
-  "fnOnEditing": function(input) {return input[0].value;},
+  "fnOnEditing": function(input) { if(input[0].value.length === 0){ input[0].value = " "; } return input[0].value;},
```

Comment:
It would be more save to write `input[0].value === ""` instead of `input[0].value.length === 0`

Why not write something like `if(input[0].value === ""){ return " "; } return input[0].value;` this would prevent the modification of `input[0].value`.